homeassistant/components/opower/strings.json

```diff
@@ -35,7 +35,7 @@
   "issues": {
     "return_to_grid_migration": {
       "title": "Return to grid statistics for account: {utility_account_id}",
-      "description": "We found negative values in your existing consumption statistics, likely because you have solar. We split those in separate return statistics for a better experience in the Energy dashboard.\n\nPlease visit the [Energy configuration page]({energy_settings}) to add the following statistics in the **Return to grid** section:\n\n{target_ids}"
+      "description": "We found negative values in your existing consumption statistics, likely because you have solar. We split those in separate return statistics for a better experience in the Energy dashboard.\n\nPlease visit the [Energy configuration page]({energy_settings}) to add the following statistics in the **Return to grid** section:\n\n{target_ids}\n\nOnce you have added them, ignore this issue."
```

Comment:
Can't we get the issue to be automatically removed when we don't detect it anymore ?

Reply:
You might have misunderstood the issue.
Historically the Opower integration has been creating a single statistic for both consumption and return to grid. If you returned to the grid it had negative values. Starting with 2025.5.0 we have a separate return to the grid statistic. And we perform a one time migration. We raise this issue so that users update their energy dashboard config and any other manual dashboard they might have had. We could check that the user has updated their energy dashboard by checking config/.storage/energy or making API changes in the energy component but it's not something worth doing.

Reply:
Do we only open the issue when there has been negative values? That way users without solar won't need to ignore an issue.

Reply:
Correct. We migrate and open the issue only if we find at least one negative value.